htmd/parameterization/dihedral.py

```diff
@@ -335,23 +287,33 @@ def _optimize_random_search(self, vector):
 
         return best_vector
 
+    def _updateDihedrals(self, parameters, keys, vector):
+        nparams = len(keys) * self.MAX_DIHEDRAL_MULTIPLICITY
+        assert vector.size == 2 * nparams + self.numDihedrals
+
+        for i, k in enumerate(keys):
+            for j, t in enumerate(parameters.dihedral_types[k]):
+                t.phi_k = vector[i*6+j]
```

Comment:
You hard coding `self.MAX_DIHEDRAL_MULTIPLICITY`

Reply:
Thanks!